src/components/Dropzone.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useCallback } from "react";
+import { useDropzone } from "react-dropzone";
+import styled, { css } from "styled-components";
+import { colors, shadows } from "../shared/constants";
+import { themeElement } from "../shared/theme";
+
+const dropStyles = css`
+  display: flex;
+  align-items: center;
+  padding: 1rem 2rem;
+  border-radius: 1rem;
+  font-family: "DM Sans", sans-serif;
+  border: none;
+  transition: box-shadow 300ms ease, background-color 300ms ease,
+    color 300ms ease;
+  cursor: pointer;
+  justify-content: center;
+  border: solid 1px ${themeElement(colors.snow, colors.ikeaBlue)};
+  box-sizing: border-box;
+
+  &:hover {
+    box-shadow: ${shadows.hover};
+  }
+`;
+
+const PrimaryDrop = styled.div`
+  ${dropStyles}
+  background-color: ${themeElement(colors.snow, colors.ikeaBlue)};
+  color: ${themeElement(colors.spaceGrey, colors.white)};
+
+  &:hover {
+    background-color: ${themeElement(colors.spaceDark, colors.white)};
+    color: ${themeElement(colors.snow, colors.spaceGrey)};
+  }
+`;
+
+const SecondaryDrop = styled.div`
+  ${dropStyles}
+  background-color: ${themeElement(colors.ikeaBlue, colors.spaceGrey)};
+  color: ${colors.white};
+
+  &:hover {
+    background-color: ${themeElement(colors.spaceDark, colors.white)};
+    color: ${themeElement(colors.snow, colors.spaceGrey)};
+  }
+`;
+
+interface DropzoneProps {
+  onUpload: Function;
+  padded?: boolean;
+  expanded?: boolean;
+  role?: "primary" | "secondary";
+  value: null | File;
+}
+
+const DropzoneComponent = ({
```

Comment:
The component should provide visual feedback to the user when a file is being dragged over it. The dragzone library provides functionality for this.

Reply:
Done